source/common/runtime/runtime_features.cc

```diff
@@ -47,6 +47,12 @@ constexpr const char* disallowed_features[] = {
     // Acts as both a test entry for deprecated.proto and a marker for the Envoy
     // deprecation scripts.
     "envoy.deprecated_features.deprecated.proto:is_deprecated_fatal",
+    "envoy.deprecated_features.bootstrap.proto:runtime",
+    "envoy.deprecated_features.redis_proxy.proto:catch_all_cluster",
+    "envoy.deprecated_features.lds.proto:use_original_dst",
+    "envoy.deprecated_features.server_info.proto:max_stats",
+    "envoy.deprecated_features.redis_proxy.proto:cluster",
+    "envoy.deprecated_features.server_info.proto:max_obj_name_len",
```

Comment:
I think we need https://github.com/envoyproxy/envoy/blob/b3f848df49233a3575d317ed3ae4f892cc5dcb76/source/common/config/utility.cc#L302 also?

Reply:
Oh man, I totally missed this in review but this should have been envoy.reloadable_features.v1_filter_json_config which is why it wasn't caught.

I tried flipping it, and it broke the bootstrap test pretty hard.  Beyond needing the same runtime layers update, there's some issue with proto constraint which I've tracked down to a really confusing red herring where when we to do proto to json conversion we end up failing and so having a completely empty proto, so failing the first validator which checks for non-empty bytes

Given this didn't land as a config change what do we think of renaming, and flipping in its own PR, ideally after some digging into the hot restart test?

Reply:
Sure whatever you think is best. I mainly want to get this deprecation moving since this will allow us to delete a ton of code (all of v1).

Reply:
Would you have cycles to look into what's going on with the filter config? I was going to ask we assign whoever introduced the variable to sorting out the test and looks like it was you :-P

Reply:
Yes, sorry, what do you want me to do exactly? If you give me a diff that is broken I can debug it. :)

Reply:
I think we land this as-is, and you flip the v1 json separately, since it currently doesn't pass tests?

Reply:
OK got it, but you want me to change the name also?